docs/overview.md

```diff
@@ -8,42 +8,68 @@ slug: /
 The Biconomy SDK is an Account Abstraction toolkit that enables the simplest UX on your dApp, wallet, or appchain.
 Built on top of [ERC 4337](https://eips.ethereum.org/EIPS/eip-4337), we offer a full-stack solution for tapping into the power of our Smart Accounts Platform, Paymasters, and Bundlers.
 
-![FullStakAA](./images/overview/fullstackaa.png)
+<details>
+  <summary>Introduction to Account Abstraction</summary>
+
+Account Abstraction aims to enhance user experience by making user accounts more flexible and functional. Instead of using an Externally Owned Account, a Smart Contract that can act as your account, powered by code instead of the Elliptic Curve Digital Signature Algorithm (ECDSA). 
```

Comment:
```suggestion
  <summary>Introduction to Account Abstraction</summary>

Account Abstraction aims to enhance user experience by making user accounts more flexible and functional. Instead of using an Externally Owned Account, a Smart Contract can act as your account, powered by code instead of the Elliptic Curve Digital Signature Algorithm (ECDSA). 
```